examples/pytorch/text-classification/run_glue.py

```diff
@@ -507,17 +507,25 @@ def compute_metrics(p: EvalPrediction):
         if data_args.task_name == "mnli":
             tasks.append("mnli-mm")
             eval_datasets.append(raw_datasets["validation_mismatched"])
+            combined = {}
 
         for eval_dataset, task in zip(eval_datasets, tasks):
             metrics = trainer.evaluate(eval_dataset=eval_dataset)
 
             max_eval_samples = (
-                data_args.max_eval_samples if data_args.max_eval_samples is not None else len(eval_dataset)
+                data_args.max_eval_samples
+                if data_args.max_eval_samples is not None
+                else len(eval_dataset)
```

Comment:
I don't see why this part is changed, could you revert?

Reply:
This went in accidentally, will be reverted. (`make style` reported a bunch of formatting changes in other files, so I've missed this one)